docs/concepts/pools/boosted.md

```diff
@@ -6,73 +6,73 @@ order: 5
 
-As explained above, these pools have Preminted BPT (the `bb-a-*` tokens) and maintain target balances for base tokens and their aToken counterparts.
+### Nesting Linear Pools
+Since BPT are ERC-20 tokens themselves, Linear Pool BPT can be nested within another pool. This creates a simple `batchSwap` path between base assets and tokens in the outer pool because traders can swap from BPT to one of the Linear Pool's underlying tokens.
 
-#### Put the Linear Pool Tokens into a Stable Pool
+## Example
 
-By adding the Linear Pool Tokens into a Stable Pool, we are effectively putting the pools inside another pool.
+Let's say we want to build a new Weighted Pool with two tokens: 50% USDC and 50% WETH. We can expect that swaps on this pool will touch just a fraction of its depth (i.e. swaps typically won't use more than ~20% of the balances in the pool since trades of that scale would significantly change the price).
+![](/images/idle-tokens.png)
+As the figure above illustrates, only a tiny amount of the pool would be touched by swaps. The remaining assets are in the pool to compose the price and reduce price impact, but are idle and therefore not generating yield or fees directly. **Boosted Pools aim to optimize assets usage of a pool.**
 
-### How do I swap between stablecoins?
+If were instead to build this pool as a Boosted Pool, the idle assets can become yield bearing versions on some hypothetical protocol, ProtocolX, with `xTokens` that are yield bearing versions of their base `tokens`. We achieve this composition by utilizing Linear Pools, the crucial building block that make Boosted Pools possible.
 
-Since Linear Pools have Preminted BPT, we can craft a `batchSwap` call that goes from one stablecoin to another.
+![](/images/linear-pool-nested.png)
 
-For example, consider the following swap steps:
+## Case Study: bb-a-USD
+![](/images/bb-a-USD.png)
 
-1. Trade USDC to the USDC Linear Pool Token
-   1. USDC -> bb-a-USDC
-2. Trade USDC Linear Pool token for the DAI Linear Pool Token
-   1. bb-a-USDC -> bb-a-DAI
-3. Trade DAI Linear Pool Token for DAI
-   1. bb-a-DAI -> DAI
+Balancer Boosted Aave USD (symbol: `bb-a-USD`) is a Composable Stable Pool that facilitates trades between three US Dollar stablecoins (`USDC`, `USDT`, and `DAI`) while sending idle liquidity to Aave. The underlying Linear Pools are:
```

Comment:
Might be nice to link to the bb-a-usd pool in the app. I've found the balance table section on the boosted pool page really helps visualize the concept to someone unfamiliar